rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Please kindly add `Getters and Setters for @book and @person instance variables ` since they are modified in the `Rental class`. Somthing like so: 
`attr_accessor :book, :person`

Reply:
Done.